docs/user-profile-and-api-key.md

```diff
@@ -0,0 +1,30 @@
+---
+id: user-profile-and-api-key
+title: User Profile & API Key
+---
+The ARGO User Profile page displays important user information including:
+- which email you have logged into the ARGO platform with
+- what your ARGO Member permission are
+- API Key management
+
```

Comment:
suggestion for above
- the email address associated with your ARGO Platform user
- your ARGO Member permissions
- API token management